src/core/net/ip6.cpp

```diff
@@ -426,6 +427,11 @@ ThreadError Ip6::HandleDatagram(Message &message, Netif *netif, int8_t interface
     {
         if (netif != NULL)
         {
+            if (otGetDeviceRole() == kDeviceRoleChild)
```

Comment:
Realm_Local_All_ThreadNodes multicast address is a realm-local unicast prefix-based multicast address, spec says 'all Thread interfaces MUST subscribe to the Realm-Local All Thread Nodes multicast address' and 'Parents MUST use IEEE802.15.4 indirect transmissions to forward multicast packets destined to realm-local all-Thread-nodes to their rx-off-when-idle Children'
Since it is a known subscribed multicast address by all thread nodes, it might be ok for the SED not registering this address to the parent.